lib/Field/FieldContent.js

```diff
@@ -14,13 +21,21 @@ export function FieldContent({ children, className, instructions }) {
         className,
         'col-span-12 md:col-span-8',
         'col-start-1',
-        'border border-solid border-neutral-90',
+        'border-solid',
         'rounded',
         'shadow',
+        'transition-all, transition-ease duration-200',
         {
           'p-6': !inStructureEditMode,
           'md:col-start-5': !inStructureEditMode,
-          'md:col-start-4': inStructureEditMode
+          'md:col-start-4': inStructureEditMode,
+          'border border-neutral-90': status === ACTIVE,
```

Comment:
💚 It might be quite clean to move the `status === ` styles to their own object.

Super 💚 comment, we could do this another time.

Reply:
Not sure I fully understand, do you mean we'd have some consts like
```
const isActive = status === ACTIVE;
const isAdded = status === ADDED;
const isDeleted = status === DELETED;
// and so on...
```